tests/metawolf/utils.py

```diff
@@ -86,7 +86,7 @@ def testStr2Bool(self) -> None:
   def testGetType(self) -> None:
     """Test that the type of a string is inferred correctly."""
     self.assertEqual(bool, utils.GetType('y'))
-    self.assertEqual(int, utils.GetType('0'))
+    self.assertEqual(bool, utils.GetType('0'))
```

Comment:
What if a recipe takes an int that can be 0?

Reply:
```
$b552a4$aws_forensics> set -p boot_volume_size 0
boot_volume_size: 50 --> 0
```

Not a problem